packages/terra-tabs/src/common-tabs/CommonTabItem.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import TabContext from './_TabContext';
+
+const propTypes = {
+  /**
+   * Key to match with the activeItemKey to handle the display of selection.
+   */
+  itemKey: PropTypes.string.isRequired,
+  /**
+   * Text to be displayed on the tab or as it's aria-label.
+   */
+  label: PropTypes.string.isRequired,
+  /**
+   * Object to be returned in the onRequestActive.
+   */
+  metaData: PropTypes.object,
+  /**
+   * Function executed during rendering to generate child content.
+   */
+  render: PropTypes.func,
+  /**
+   * @private
+   * The id of the tab.
+   */
+  id: PropTypes.string,
+  /**
+   * @private
+   * The id of the panel associated to the tab.
+   */
+  associatedPanelId: PropTypes.string,
+  /**
+   * @private
+   * The indicator whether or not the tab content is active.
+   */
+  isActive: PropTypes.bool,
+  /**
+   * @private
+   * The html element to be hold the portaled item content.
+   */
+  portalElement: PropTypes.instanceOf(HTMLElement),
+  /**
+   * Indicates if the pane should be disabled.
+   */
+  isDisabled: PropTypes.bool,
+};
+
+/**
+ * The CommonTabItem component represents a tab within the CommonTabs, defining
+ * its labels and content. The CommonTabItem component must only be used as an
+ * immediate child of the CommonTabs component.
+ */
+const CommonTabItem = ({
+  id,
+  icon,
+  customDisplay,
+  associatedPanelId,
+  label,
+  render,
+  portalElement,
+  isIconOnly,
```

Comment:
`isIconOnly` is missing in props description

Reply:
Added https://github.com/cerner/terra-framework/pull/1678/commits/8b40dac988c07e97c671313470b8975ddf258da8